src/main/scala/viper/gobra/translator/encodings/PermissionEncoding.scala

```diff
@@ -44,8 +44,20 @@ class PermissionEncoding extends LeafTypeEncoding {
       case (e: in.DfltVal) :: ctx.Perm() / Exclusive => unit(withSrc(vpr.NoPerm(), e))
       case fp: in.FullPerm => unit(withSrc(vpr.FullPerm(), fp))
       case np: in.NoPerm => unit(withSrc(vpr.NoPerm(), np))
-      case fp@ in.FractionalPerm(l, r) => for {vl <- goE(l); vr <- goE(r)} yield withSrc(vpr.FractionalPerm(vl, vr), fp)
       case wp: in.WildcardPerm => unit(withSrc(vpr.WildcardPerm(), wp))
+      case co@ in.Conversion(_, exp) => for { e <- goE(exp) } yield withSrc(Function.untupled(e.withMeta), co)
```

Comment:
I am bit confused. Is this the general conversion node?
If yes, then I am surprised that tests using conversions currently work. The FinalTypeEncoding should check that nodes are supported at most once, but this case seems to match on all conversion nodes, not just the conversion that result in an expression of type perm.